man/rgbasm.5

```diff
@@ -369,63 +369,93 @@ equals $roman clz ( n )$.
 delim off
 .EN
 .Ss Fixed-point expressions
-Fixed-point numbers are basically normal (32-bit) integers, which count fractions instead of whole numbers.
-They offer better precision than integers but limit the range of values.
-By default, the upper 16 bits are used for the integer part and the lower 16 bits are used for the fraction (65536ths).
-The default number of fractional bits can be changed with the
+Fixed-point numbers are technically just integers, but conceptually they have a decimal point at a fixed location (hence the name).
+This gives them increased precision, at the cost of a smaller range, while remaining far cheaper to manipulate than floating-point numbers (which
+.Nm
+does not support).
+.Pp
+The default precision of all fixed-point numbers is 16 bits, meaning the lower 16 bits are used for the fractional part; so they count in 65536ths of 1.0.
+This precision can be changed with the
 .Fl Q
-command-line option.
-You can also specify a precise fixed-point value by appending a
+command-line option, and/or by
+.Ic OPT Q
+.Pq see Sx Changing options while assembling .
+An individual fixed-point literal can have its own precision overridden by appending a
 .Dq q
-to it followed by the number of fractional bits, such as
-.Ql 12.34q8 .
+followed by the number of fractional bits: for example,
+.Ql 1234.5q8
+is equal to $0004d280
```

Comment:
May as well set a good example of how to use `_`.
```suggestion
is equal to $4d2_80
```

Reply:
I padded it to highlight that the entire number is still 32-bit. What do you prefer between this and
```suggestion
is equal to $0004d2_80
```
?

Reply:
`$0004d2_80` looks good!